src/pharmpy/tools/resmod/results.py

```diff
@@ -17,13 +16,15 @@ def calculate_results(base_model, iiv_on_ruv, power):
     base_ofv = base_model.modelfit_results.ofv
     dofv_iiv_on_ruv = iiv_on_ruv.modelfit_results.ofv - base_ofv
     dofv_power = power.modelfit_results.ofv - base_ofv
+    omega_iiv_on_ruv = iiv_on_ruv.modelfit_results.parameter_estimates[-2]
```

Comment:
Even though it would work in this case I wouldn't use numerical indices. Better is to index directly using the name of the parameter

Reply:
Thanks for fixing this.